VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionComment.java

```diff
@@ -110,6 +111,15 @@ public boolean isAbuseFlagged() {
         return editableFields;
     }
 
+
+    public int getChildCount() {
+        return childCount;
+    }
+
+    public void setChildCount(int childCount) {
+        this.childCount = childCount;
+    }
```

Comment:
This could be changed to a simple increment method, since that's the only way we should ever need to change the child count.